beluga/include/beluga/views/sample.hpp

```diff
@@ -183,6 +188,10 @@ struct sample_fn {
  * To use this, the input range must model the
  * [random_access_range](https://en.cppreference.com/w/cpp/ranges/random_access_range)
  * and [sized_range](https://en.cppreference.com/w/cpp/ranges/sized_range) concepts.
+ *
+ * This view implements multinomial resampling for a given range of particles. The core idea
+ * is to generate independently N random numbers and use them to select particles from the
+ * input range.
```

Comment:
@nahueespinosa meta: independent as in statistically independent :thinking:? Isn't it the same random variable $I \sim Mn(p_1, ..., p_k)$ of indices / iterators to the range? Perhaps you meant: 
```suggestion
 * This view implements multinomial resampling for a given range of particles.
 * The core idea is to draw random indices / iterators to the input particle range
 * from a [multinomial distribution](https://en.wikipedia.org/wiki/Multinomial_distribution)
 * parameterized after particle weights (and assumed uniform for non-weighted particle ranges).
```

Reply:
> independent as in statistically independent 🤔?

Independent compared to other resampling methods like systematic resampling or residual resampling.
https://bisite.usal.es/archivos/resampling_methods_for_particle_filtering_classification_implementation_and_strategies.pdf

But your suggestion looks great!